classes/simplemap.php

```diff
@@ -613,17 +613,10 @@ function enqueue_frontend_scripts_styles() {
 			global $post;
 			$options = $this->get_options();
 
-			// Rewrite rules if we've changed the location permalink slug.
-			if ( get_option( 'sm-rewrite-rules' ) ) {
-				global $wp_rewrite;
-				$wp_rewrite->flush_rules();
-				delete_option( 'sm-rewrite-rules' );
-			}
-
 			// Frontend only.
 			if ( ! is_admin() && is_object( $post ) || apply_filters( 'sm-force-frontend-js', '__return_false' ) ) {
 				// Bail if we're not showing on all pages and this isn't a map page.
-				if ( ! in_array( $post->ID, explode( ',', $options['map_pages'] ) ) && ! in_array( 0, explode( ',', $options['map_pages'] ) ) ) {
+				if ( ! in_array( $post->ID, explode( ',', $options['map_pages'] ) ) && ! in_array( 0, explode( ',', $options['map_pages'] ) ) && !is_singular( 'sm-location' ) && empty( $_GET[ 'sm_map_iframe' ] ) ) {
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed
* Expected 0 spaces after opening bracket; 1 found
* Space after opening parenthesis of function call prohibited
* Expected 0 spaces before closing bracket; 1 found
* Line exceeds 120 characters; contains 215 characters

<!-- Comment by [SideCI](https://sideci.com) -->